nbclient/client.py

```diff
@@ -507,19 +584,25 @@ def _check_alive(self):
             self.log.error("Kernel died while waiting for execute reply.")
             raise DeadKernelError("Kernel died")
 
-    async def _wait_for_reply(self, msg_id, cell=None):
+    def _wait_for_reply(self, msg_id, cell=None):
+        # Needed by papermill until it updates to consume async_wait
+        loop = get_loop()
+        return loop.run_until_complete(
+            self.async_wait_for_reply(msg_id, cell=cell)
+        )
+
+    async def async_wait_for_reply(self, msg_id, cell=None):
```

Comment:
Should it be `_async_wait_for_reply` for name consistency with `_wait_for_reply`?

Reply:
I was thinking of exposing `async_wait_for_reply` as not private so it could be used in contracts for extensibility since papermill ended up needing the blocking verison to extend behavior. If I'm doing that I should probably commit and make `wait_for_reply`, or revert the decision overall and keep with `_async_wait_for_reply`.